source/command_generators/linkers.d

```diff
@@ -33,7 +33,18 @@ string[] parseLinkConfiguration(immutable BuildConfiguration b, OS target, Compi
             commands ~= "--format=default";
             commands ~= "rcs";
             commands ~= buildNormalizedPath(outputDirectory, getOutputName(targetType, name, os));
-            commands ~= getLinkFiles(b.sourceFiles);
+            
+            import std.file;
+            import std.algorithm: canFind;
+            
+            foreach (DirEntry key; dirEntries(b.workingDir, SpanMode.shallow))
```

Comment:
That is super hacky, at this step, it should not be looking for linker files anymore.

Reply:
Yeah but for some reason the array returned by `getLinkFiles` is empty.

Reply:
So I took this approach to deliver something